apis/v1/events/schema.yaml

```
@@ -331,6 +331,70 @@ paths:
           $ref: 'http://localhost:3320/commons/components#/components/responses/404'
         '500':
           $ref: 'http://localhost:3320/commons/components#/components/responses/500'
+  "/api/processed-messages/{idempotencyKey}":
+    post:
+      tags:
+      - Mensagens
+      summary: Reenvie uma mensagem processada a partir do seu id de idempotência.
+      description: Reenvie uma mensagem processada.
+      operationId: ResendProcessedMessageAsync
+      parameters:
+      - name: idempotencyKey
+        in: path
+        required: true
+        schema:
+          minLength: 1
+          type: string
+          format: uuid
+      - name: api-version
+        in: header
+        schema:
+          type: string
+      responses:
+        '200':
+          description: Recurso encontrado
+          headers:
+            x-bkly-correlation-id:
+              description: Id correlação da operação requisitada.
+              schema:
+                type: string
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/ProcessedMessageSearchResponse"
+        '400':
+          description: Há algum problema com a sua requisição.
+          headers:
+            x-bkly-correlation-id:
+              description: Id correlação da operação requisitada.
+              schema:
+                type: string
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/ErrorResponse"
+        '404':
+          description: Recurso não encontrado
+          headers:
+            x-bkly-correlation-id:
+              description: Id correlação da operação requisitada.
+              schema:
+                type: string
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/ErrorResponse"
```

Nós já temos estes componentes na pastinha commons 😉

```suggestion
                "$ref": "http://localhost:3320/commons/components#/components/responses/404"
```